example/android/build.gradle

```diff
@@ -28,6 +28,7 @@ task fetchDependencies() {
 buildscript {
     def androidTestAppDir = "../../node_modules/react-native-test-app/android"
     apply from: "$androidTestAppDir/dependencies.gradle"
+    apply from: "${androidTestAppDir}/test-app-util.gradle"
```

Comment:
This is no longer needed.
```suggestion
```

Reply:
Done in https://github.com/react-native-async-storage/async-storage/pull/910/commits/2e939383f30c9878ad1b23831e666c9663452608